basic/src/test/java/org/jboss/hal/testsuite/test/configuration/singleton/SingletonSubsystemOperations.java

```diff
@@ -0,0 +1,51 @@
+package org.jboss.hal.testsuite.test.configuration.singleton;
+
+import org.apache.commons.lang.RandomStringUtils;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+import java.io.IOException;
+
+/**
+ * Class containing common methods for singleton subsystem tests
+ */
+public class SingletonSubsystemOperations {
+
+    private Operations operations;
+
+    /**
+     * Create new instance
+     * @param operations used instance of operations
+     * @return new instance
+     */
+    public static SingletonSubsystemOperations withOperations(Operations operations) {
+        return new SingletonSubsystemOperations(operations);
+    }
```

Comment:
Please don't use this pattern for Operations classes, it makes inconsistent usage for them and makes it rather confusing than helping.